File: src/backend/Softtek.Application/DTOs/PerguntaDto.cs
using NUlid;

namespace Softtek.Application.DTOs
{
    public class PerguntaDto
    {
        public Ulid Codigo { get; set; }
        public string Descricao { get; set; }
        public bool Desativado { get; set; }
        public Ulid EscalaCodigo { get; set; }
        public string EscalaDescricao { get; set; }
        public List<EscalaValorDto> ValoresAceitos { get; set; }
    }

    public record NovaPerguntaDto(
        Ulid BlocoCodigo,
        string Descricao,
        Ulid EscalaCodigo
    );
}

File: src/frontend/app/src/main/java/com/fiap/softtekapp/data/model/PerguntaDto.kt
package com.fiap.softtekapp.data.model

import com.fiap.softtekapp.data.model.EscalaValorDto

data class PerguntaDto(
    val codigo: String,
    val descricao: String,
    val desativado: Boolean,
    val escalaCodigo: String,
    val escalaDescricao: String,
    val valoresAceitos: List<EscalaValorDto>
)